game_of_life.py

```diff
@@ -0,0 +1,100 @@
+
+"""A Conway's Game of Life board class.
+
+This module provides a basic board class that tracks
+the state of the board (between generations). The board
+class provides the methods for getting a cells live
+neighbor count and updating a cells value.
+
+"""
+
+import numpy as np
+
+class Board():
+    """Conway's Game of Life board."""
+
+    def __init__(self, size):
+        self.board = np.random.randint(2, size=size)
+        self.north_transform = (0, -1)
+        self.south_transform = (0, 1)
+        self.east_transform = (1, 0)
+        self.west_transform = (-1, 0)
+    
+    def get_cell_value(self, cell_coordinates):
+        """Return cell value for given coordinates."""
+        return self.board[cell_coordinates] # ndarrays take tuple indicies
+
+    def transform_coordinate(self, coord_one, coord_two):
+        """Return new cell coordinates after applying coordinate transformations."""
+        return tuple(map(lambda x,y: x + y, coord_one, coord_two))
+
+    def get_neighbor_coord(self, cell_coordinates, direction):
+        """Return Moore neighbor coordinates for given direction.""" 
+        if direction == 'n':
```

Comment:
It might be simpler/less verbose to write something like:
```
neighbors = []
for direction in [(1,0), (0, -1)...]:
     neighbors.append(transform_coordinate(cell_coordinates, direction)
return neighbors
```

Reply:
@lbernick thank you! Yes, that is much cleaner than all of these elif statements. I'll implement your suggestion.